### ISSUE TITLE
Lost my wallet after reopen my computer normally

### ISSUE THREAD

**Issue by Bener3** [MetaMask/metamask-extension#3191]
Today, I opened my computer normally but when i open google chrome and click on the metamask, I cant see my account. There was a new account open screen. I did not have my private key. Only seed phrase. How can i find the private key from my computer? Its operating system is MAC OSX. Can you please help me, i am so panicked. I checked my address from etherscan. Its normal. My balance is still the same.

**Comment by Bener3** [comment]
By the way, I thought its normal to have that new account screen, and accept the terms and agreement. And it created a new account. Now I cannot see my old account. How can it be possible? I did not even turn off my computer, It was on a sleep mode. Please somebody from support help me. I have my seed phrase

**Comment by docloulou** [comment]
Click on "Import Existing DEN" just after accepting Terms... and copy paste your backup seed

**Comment by Bener3** [comment]
@docloulou I have created another account. How can i go back to ''accepting terms'' screen?

**Comment by docloulou** [comment]
Uninstall metamask then reinstall it

**Comment by bvand** [comment]
The same thing happened to me today. I was using an account I previously had access to as recently as yesterday.

Today, when I tried to open MetaMask, I got the Privacy Notice, Terms of Use update and then a screen asking to encrypt my new Den (see screenshots below). This is strange because I was always able to just login with my password.

When I recover with my seed phrase, I'm unable to view the account I had access to before. Here is the account on etherscan: https://etherscan.io/address/0xaba1c2fd42df0af1ffed00833da1e71536a37fd4

Any help would be appreciated here. This wallet is the only one I've made with MetaMask, so I don't understand how restoring it could lead to me losing access.

<img width="357" alt="screen shot 2018-02-05 at 10 46 03 pm" src="https://user-images.githubusercontent.com/6099457/35840860-a9956776-0ac6-11e8-9549-50c3bf42fc99.png">
<img width="360" alt="screen shot 2018-02-05 at 10 45 50 pm" src="https://user-images.githubusercontent.com/6099457/35840861-a99f75fe-0ac6-11e8-815d-880018becf37.png">

**Comment by frankiebee** [comment]
@docloulou you can logout through the menu in the top right corner and restore from seed phrase their.

@bvand are the 2 accounts different? or is it that you are just missing the transaction history?

**Comment by frankiebee** [comment]
oops sorry i meant @Bener3 you can logout through the menu in the top right corner and restore from seed phrase their.

**Comment by Bener3** [comment]
I restored my old account although I have created a new one mistakenly. This is the message that i got from metamask support ; 

""To restore with your seed phrase is quite easy. You can start from one of two screens. Either the first screen you see after accepting terms and conditions of the setup process. This link is titled "Restore from existing DEN." Or you can restore from the "lock" screen, which is when you're logged out. This link says "Restore from seed phrase."

From there, just paste your seed phrase exactly as it was presented to you. No extra spaces, no capital letters. Just plainly how it was presented. 

To restore with your seed phrase is quite easy. You can start from one of two screens. Either the first screen you see after accepting terms and conditions of the setup process. This link is titled "Restore from existing DEN." Or you can restore from the "lock" screen, which is when you're logged out. This link says "Restore from seed phrase."

From there, just paste your seed phrase exactly as it was presented to you. No extra spaces, no capital letters. Just plainly how it was presented. 

If you have multiple ETH addresses created under one seed phrase, do the following once you complete the above steps.

After restoring with your seed phrase you must click the "Create Account" button from the drop down menu - for each old account you had in your original installation you must add a new account. Currently we don't re-add them automatically. But if you follow this step it will add your other accounts in the order they were originally created. ""


Hope it works for others too.

**Comment by bvand** [comment]
@frankiebee the accounts are different. 

When I create additional accounts as @Bener3 did, none of them match as well.

**Comment by Bener3** [comment]
@bvand Ofc. I reopened it. Sorry for being late.

**Comment by Bener3** [comment]
@bvand Try it again. I got the same error in my first try. Try 2-3 times at least. I had the same password for both old account and new account(which has opened by me mistakenly). Try to use the old account's seed phrase when you are restoring, NOT THE NEW ONE'S SEED PHRASE.

**Comment by bvand** [comment]
@Bener3 Thanks.

I've restored from my seed phrase multiple times - I only ever had one account, and I used that seed phrase.

**Comment by bvand** [comment]
@frankiebee this problem has persisted, I've even tried using the version of Metamask that I first installed and restoring my seed phrase with that, but I still don't have access to my account.

I'm positive that I only ever created one Metamask wallet, and that is the one I have the seed phrase for. The file that I saved has the correct address. It is in this format:
```
Account Address: 0xaba1c2fd42df0af1ffed00833da1e71536a37fd4
Account Pwd: <pwd>

MetaMask
Account Encryption Pwd: <pwd>
Recovery Phrase: <seed>
```

Please let me know if there's any additional information that I can provide that would be helpful.

**Comment by danfinlay** [comment]
Hi, very sorry for the unsuccessful help you got so far.

To restore an account from seed phrase, you don't have to uninstall/reinstall. You can simply lock metamask (top right menu) and click "Restore from seed phrase".

When you restore from the seed phrase, we only re-generate your first account at first, but you can click "Create account" in the top right account menu extra times to re-generate and restore any additional accounts.

This should solve your problem, let us know if it doesn't.

**Comment by bvand** [comment]
@danfinlay unfortunately that doesn't solve the problem. I logged out and restored from the seed phrase, no change. The first account that gets generated is https://etherscan.io/address/0xd6b948c516ff25b6d7067f8d7de147dc1ed2672b when I expect it to be https://etherscan.io/address/0xaba1c2fd42df0af1ffed00833da1e71536a37fd4 (the account specified in the file that also has my seed phrase).

None of the additional addresses that I generate after restoring from seed match the one that has my tokens either.

**Comment by danfinlay** [comment]
Sorry @bvand, what you've provided is basically inexplainable by the current software unless either:

- There was user error
- There was a bug in our code

We regularly audit our code to prevent this possibility, and [keep a bug bounty open](https://github.com/MetaMask/metamask-extension/issues/3127) for anyone who can reproduce an issue where the seed phrase that is first given to a person does not re-generate their original account, because people occasionally claim this happened. We have no way of knowing whether these are valid claims or not.

I would keep the phrase you wrote down in case an issue is ever discovered that might allow re-generating from what you have, but for now I have to say this looks unexplained, and I'm very sorry it happened, but there's nothing we can do at this point to help.

I know it doesn’t help, but our terms do make it clear that how you use our software is your own responsibility. I'm sorry if you used it for something of value without first confirming you were able to restore your account with the seed phrase that you backed up. I would recommend doing that in the future, to avoid issues like this.

Again, I'm deeply sorry, because I cannot know if I wrote a bug that caused this, but I wish you the best of luck.

**Comment by bvand** [comment]
@danfinlay thanks for the response, I understand. I'll definitely make sure to confirm first next time - big miss there.

The frustrating thing is that the account address in my exported seed file matches the one that I (previously) had access to... so I have no reason to think I did anything wrong here.

Separately, I don't understand why MetaMask would suddenly require that I restore from the seed phrase rather than use my password. The problem first started when I suddenly was unable to login with my password. Was that by design for the new Terms of Use? Or was it because my browser cache was maybe cleared?

**Comment by danfinlay** [comment]
We've recently been made aware that browsers sometimes wipe the type of storage we were using. We have a pending fix for this issue that will make browsers lose data less often, but even if they lose it less often, an incorrectly backed up seed phrase would eventually result in the same result (worse, maybe, because you'd trust it longer), so I'm not sure that will be a big improvement for users who seem to have backed up the wrong phrase.

Again, I'm very sorry.

**Comment by bvand** [comment]
Yeah I certainly did trust it for a while - I think I first started using MetaMask around August.

Well, I appreciate your responses here. Let this be a cautionary tale... always test your seed phrase immediately.

**Comment by BGzetro** [comment]
I had the exact same problem and have lost alot of money as a result :(

Seems like this has occurred to quite a few people lately.

https://github.com/MetaMask/metamask-extension/issues/3258

**Comment by noahbreal** [comment]
Same thing today , Meta mask had me accept some terms of use then it was like I never had a wallet, it asked me to create or import. So I imported with seed phrase and got a new set of addresses that do not correspond to mine. Luckily I have second installation on FIrefox but kind of scary.

**Comment by bvand** [comment]
@danfinlay can you reopen this issue? seems like I didn't make a mistake after all, if @BGzetro and @noahbreal are having the same issue.

**Comment by danfinlay** [comment]
This issue is a duplicate of #3127, so there's no reason to reopen it. We will continuously raise bug bounties as long as bugs are claimed to exist, but currently no one has been able to find a way that this could have happened to you with the current version of MetaMask.

**Comment by lazaridiscom** [comment]
(notes for myself, just ignore)

> We've recently been made aware that browsers sometimes wipe the type of storage we were using. We have a pending fix for this issue that will make browsers lose data

=> Verify Conceptual Constructs

> Today, when I tried to open MetaMask, I got the Privacy Notice, Terms of Use update and then a screen asking to encrypt my new Den (see screenshots below).

=> Terminology, Process

> Same thing today , Meta mask had me accept some terms of use then it was like I never had a wallet, it asked me to create or import. So I imported with seed phrase and got a new set of addresses that do not correspond to mine. Luckily I have second installation on FIrefox but kind of scary.

=> SEVERE, CRITICAL

> can you reopen this issue? seems like I didn't make a mistake after all,

=> Assess who has priority in the public issue-tracker: the users or the developers (personal assessment: users)

**Comment by JACC35** [comment]
This is exactly what happened to me yesterday, cleared my normal login screen and is only allowing me to create new DEN... only thing is, I can't find my seed words.  Is there any way to restore wallet without that?  I found old doc I had saved with different account number and private key along with the public address that's associated with all of my tokens. I'm in a panic, can anyone help this tech savvy-less user?

**Comment by danfinlay** [comment]
@JACC35 
> I can't find my seed words. Is there any way to restore wallet without that?

I'm sorry, but there is no way to restore your accounts without either your seed words or a backup of the computer when the accounts were working.

We're always working hard to make it harder for users to not back up their words, and we are also always working to make the application more resilient to disk errors, but sometimes computers still have issues, and so it is still the user's responsibility to keep their original seed words backed up.

I hope you find a backup that you wrote down.

**Comment by JACC35** [comment]
So if I backed up my computer would a system restore back to a couple days ago restore the wallet? I just can't find my seed words for this wallet but I've found that account number with private key... I only have one Metamask on this computer but I may be misunderstanding.. would the account number with private key be linked to the public address where my tokens are? why would it all of a sudden clear my login plugin and only give me the option of creating a new one?  Thanks for responding...

**Comment by danfinlay** [comment]
Yes, a system restore would also restore the wallet. You can email us for more detailed support at support@metamask.io

**Comment by JACC35** [comment]
@danfinlay thanks,  I just can't remember if I wrote down my seed words or if I chose the option to save to file.. if I chose the option to save as file, what format would it be in so I can search for it? Just Word doc?  It was almost a year ago and I just can't remember... since I was so new to this I saved or printed out EVERYTHING, so it may be saved, I just don't know what format to search... it's not like me not to have the seed words written down... I will definitely try the system restore, hopefully that will work.  I've contacted support since yesterday several times but have only received automated responses that my email was received... but that other question I had - the public wallet address that my tokens are stored, that would be linked to this account number with private key that I have?

**Comment by JACC35** [comment]
And thanks so much for responding, you're the only one I've gotten any reply from

**Comment by JACC35** [comment]
@danfinlay I've located the .json file but when I enter the only password I've ever had for Metamask, an error message comes up saying passphrase is incorrect and prevents me from importing.  I don't understand why it won't accept..?  I've tried emailing support@metamask several times now and I'm not getting any replies back for the last couple days.  The system restore didn't work either.

**Comment by danfinlay** [comment]
What was your issue number on our support system? We didn't think we were missing any.

You might want to look inside that JSON file. It should have a `data` key, and a `keyringController` key that has a lot of data in it. Do you see that? If it doesn't have data, then there's nothing to decrypt. Sorry that tool isn't very refined.

**Comment by JACC35** [comment]
#11421 and 11427 but I see an email from James now which I just replied to.  Will I get a response that way?  I'm pretty sure that's the correct json file..  I'm not giving up, have to get this access back.  Thanks @danfinlay

**Comment by danfinlay** [comment]
Yes, James will help you on the support system.

**Comment by danfinlay** [comment]
This issue is fixed by https://github.com/MetaMask/metamask-extension/pull/3083 and will be solved in the next MetaMask release `4.3.0+`.
